adalib/ada/datasets/multisource.py

```diff
@@ -73,7 +64,7 @@ def __init__(
         Args:
             source_access (DatasetAccess): [description]
             target_access (DatasetAccess): [description]

```

Comment:
`source_access` and `target_access` also lack a description.

Reply:
Done